localisation/00_wc_mnm_replace.csv

```diff
@@ -2,6 +2,5 @@
-devil_worshipers_dark_healing_power_has_mystics_dlc_true_key;Power: �YDark Healing�!\nEnables �Gright-clicking�! on your character's portrait in order to perform a dark ritual to heal yourself, in exchange for �Y[Root.Society.GetCurrency]�!.;Pouvoir: �YGu�rison d�moniaque�!\nPermet d'ex�cuter un rituel mal�fique pour vous soigner, en �change de �Y[Root.Society.GetCurrency]�!.\n(Faites un clic droit sur votre portrait);Macht: �YDunkle Heilung�!\nErm�glicht das Rechtsklicken auf das Portr�t Eures Charakters, um f�r �Y[Root.Society.GetCurrency]�! ein dunkles Ritual zu Eurer eigenen Heilung durchzuf�hren.;;Poder: �YCuraci�n oscura�!\nActiva el clic con el bot�n secundario en el retrato de tu personaje para llevar a cabo un ritual oscuro para curarte a cambio de �Y[Root.Society.GetCurrency]�!.;;;;;;;;;x
-;;;;;;;;;;;;;;
\ No newline at end of file
+DW_prolong_life_power_has_mystics_dlc_true_key;Power: �YAbsorb Lifeforce�!\nEnables �Gright-clicking�! the portraits of your children in an attempt to steal their life force, in exchange for �Y[Root.Society.GetCurrency]�!.;Pouvoir : �YAbsorber la force vitale�!\nPermet de voler la force vitale de vos enfants, en �change de �Y[Root.Society.GetCurrency]�!.;Macht: �YLebenskraft absorbieren�!\nErm�glicht das Rechtsklicken auf die Portr�ts Eurer Kinder, um zu versuchen, ihnen f�r �Y[Root.Society.GetCurrency]�! die Lebenskraft zu rauben.;;Poder: �YAbsorber fuerza vital�!\nActiva el clic con el bot�n secundario en los retratos de tus hijos para intentar arrebatarles la fuerza vital a cambio de �Y[Root.Society.GetCurrency]�!.;;;;;;;;;x
+devil_worshipers_dark_healing_power_has_mystics_dlc_true_key;Power: �YDark Healing�!\nEnables �Gright-clicking�! on your character's portrait in order to perform a dark ritual to heal yourself, in exchange for �Y[Root.Society.GetCurrency]�!.;Pouvoir : �YGu�rison d�moniaque�!\nPermet d'ex�cuter un rituel mal�fique pour vous soigner, en �change de �Y[Root.Society.GetCurrency]�!.;Macht: �YDunkle Heilung�!\nErm�glicht das Rechtsklicken auf das Portr�t Eures Charakters, um f�r �Y[Root.Society.GetCurrency]�! ein dunkles Ritual zu Eurer eigenen Heilung durchzuf�hren.;;Poder: �YCuraci�n oscura�!\nActiva el clic con el bot�n secundario en el retrato de tu personaje para llevar a cabo un ritual oscuro para curarte a cambio de �Y[Root.Society.GetCurrency]�!.;;;;;;;;;x
```

Comment:
Are you sure about this?
It's vanilla loc.

Reply:
are we making replacement files for the individual vanilla files now?

Reply:
Well, IIRC, it was created before current replacement file.
Also, this is not replacement. It's mod lines with vanilla localization.